opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/WebTracingProperties.java

```diff
@@ -22,4 +28,21 @@ public Pattern getSkipPattern() {
     public void setSkipPattern(Pattern skipPattern) {
         this.skipPattern = skipPattern;
     }
+
+    public List<String> getUrlPatterns() {
+        return urlPatterns;
+    }
+
+    public void setUrlPatterns(List<String> urlPatterns) {
```

Comment:
Can this be specified in spring config e.g. yml?

Reply:
Yes, that's the main point.
```
opentracing:
  spring:
    web:
      url-patterns:
        - /*
        - ...
```

Reply:
@pavolloffay the test is in place to ensure the list is loaded correctly (I added application-test.yml)

Reply:
@jfbreault I could make it work only like:

`urlPatterns: /a, /b`

https://github.com/opentracing-contrib/java-spring-web/pull/63/files#diff-2321c00ebd91c85cf1819d2606f08147R4

Reply:
My exemple seem wrong, the `-` need to be on the same indent as the parent so something like this:
```
opentracing:
  spring:
    web:
      url-patterns:
      - /*
      - ...
```